ADMIN : OPPORTUNITY PANEL DONE

Sheetal-04 opened the pull request:
Note : On my render steps when the user go to the next step the last step background changes but because of color issue that is not visible only.

For Better review :  

In Eligibility Criteria : "Candidates for a Software Development Engineer role should demonstrate technical proficiency in relevant programming languages, strong problem-solving abilities, effective communication and collaboration skills, and an aptitude for adaptability and continuous learning." fill little long content  . (Something  like this)

Sheetal-04 committed:
ADMIN : OPPORTUNITY PANEL DONE

AmanGupta2626 commented:
<img width="846" alt="Screenshot 2024-04-10 at 2 13 22 PM" src="https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/87798432/0b681d3d-c426-4a2c-ad7e-396e37e8c4fe">
please look into the del icon and that white circle that is not looking good to me

AmanGupta2626 commented:
<img width="393" alt="Screenshot 2024-04-10 at 2 19 30 PM" src="https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/87798432/d65c4d8e-cfca-4deb-8e6c-35be6c430e24">
i think you should  remove - from OFF-LINE

AmanGupta2626 commented:
<img width="373" alt="Screenshot 2024-04-10 at 2 21 22 PM" src="https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/87798432/5bcf0534-14de-45ae-ae91-87b367ee7b1a">
You can increase the size of back button

Nandani-Paliwal reviewed (changes requested):
- In the placeholder, in place of `Enter Opportunity Role`, make it like `e.g., Backend Developer`
Similarly for every input element, so that the admin got an idea about the fields
- Remove the shadow from the input element, not looking good in pov
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/632b026d-f506-4921-9b48-60f82e87bea1)
- Change this to `Do you want to publish this opportunity ?`
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/cfb9d376-709d-47a0-b8c6-37308de0d926)
- Align them properly and change the color of delete icon to red
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/eb6715be-76b9-4250-a1c5-ef58c94baad2)
- If we navigate through the browser back button then we are redirecting to the `my-profile` page.
- This field should accept only links 
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/87b18eed-55ed-4a27-be8b-12d5e01bf168)
- Similarly, the `opportunity package` field should accept only positive numbers
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/98f9591c-ec2e-430d-8531-a0e1f08997cb)

- Here, make  the bg of inactive tab in the shade of light gray for light theme, it is looking good for dark theme, also the bg of should be green once it done( I know that is a bug, you have done but it not working for any reason, fix it, just mentioning here to have record )
![Screenshot from 2024-04-10 21-05-49](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/10f1b3fb-cc3f-4b30-9be6-8cb115e23de7)
- Both the buttons should be identical, in shape and size
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/ffb451d0-d02b-4a9e-8ba3-49b2cca014d1)
- Make the buttons center alitgn for `md` and small screen size, and maybe we can make them equal in width and height or just swap the position of the two.
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/bd274ed2-842f-41ba-adae-2026bebeac99)
 
- `Shaping Sucess` should come in same line, it is looking quite odd, I have added the screenshot with the dimension so that it become easier for you track .
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/5f710627-ec1d-4d91-aed6-66f0837a9cdb)

- Also, what if we apply `font-semibold` and text-base` in the label, ig it will look good.
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/0de480b5-aa1b-4a7a-86db-942710eb88d3)
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/c23e6049-5d6e-4a06-89ca-e6d2a8c9521c)

Nandani-Paliwal commented on src/global.css:
```diff
@@ -29,4 +29,14 @@
\ No newline at end of file
+}
+
+.input-style {
```
Why we have added the css  in the global.css also when tailwind classes is only used

Sheetal-04 replied:
Used tailwind classes !! For all the inputs in the entire project same classes are used so instead of writing these classes again & again i just gave the whole class a name & that is better to give the input only classname input-style (Which increase code-redability ,code-reusability and better maintainance).

Sheetal-04 commented:
> * In the placeholder, in place of `Enter Opportunity Role`, make it like `e.g., Backend Developer`
=>DONE ✅

> * Similarly for every input element, so that the admin got an idea about the fields
=>DONE ✅

> * Remove the shadow from the input element, not looking good in pov
=>📌 Without shadow the input is not looking nice in dark theme shadow is giving kind of base to that input still for correction i have reduced the shadow from **_shadow-white/50_** to **_shadow-white/25_**
>   ![image](https://private-user-images.githubusercontent.com/83964826/321297870-632b026d-f506-4921-9b48-60f82e87bea1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEyOTc4NzAtNjMyYjAyNmQtZjUwNi00OTIxLTliNDgtNjBmODJlODdiZWExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZkMWViM2NlNDNlYTQyZTUwOTJjZTJjMGIxYWViZmRjMjk5NTJlMTgyYmUzNTRmMzU4NDIzMjg4NDAwODA4NjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.L-6MUAvyET2kp1ZBlz0r8jjoUEafVKzubAcHnzMYlCc)
> * Change this to `Do you want to publish this opportunity ?`
=>DONE ✅

>   ![image](https://private-user-images.githubusercontent.com/83964826/321298857-cfb9d376-709d-47a0-b8c6-37308de0d926.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEyOTg4NTctY2ZiOWQzNzYtNzA5ZC00N2EwLWI4YzYtMzczMDhkZTBkOTI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlYzQ3MTRhNzVmMDRiODcxNDg0YTIxODI5NWNjNDNlMmRiMDA0NWVhZDYzNmVhY2ZiNzg2NTg0YjZjNWE3NDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bQcEhvfizPhYvWi7jvjA5-ansPhsxW8uGQQhXBo0hBw)
> * Align them properly and change the color of delete icon to red
=>(ICON->RED) DONE ✅ (📌Alignment & button issue is in working!! )

>   ![image](https://private-user-images.githubusercontent.com/83964826/321300351-eb6715be-76b9-4250-a1c5-ef58c94baad2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDAzNTEtZWI2NzE1YmUtNzZiOS00MjUwLWExYzUtZWY1OGM5NGJhYWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzYWNhNDI5NGM0YTQ5YjhlNWYwMzYwYTEyNmM5ZjdlYzc4YWYwNTU4MTU3OGQ2MjVkMjFkMWJkMmM1Y2U3ODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YN7F2qqtlkdgDmRJa-sNFxmuFe2eHb0R5I2xXpG8PA0)
> * If we navigate through the browser back button then we are redirecting to the `my-profile` page.
=>📌That is because the Back button is going back to the component which is rendering on the basis of the step &  all the components on the same path !! There is nothing which is connected to the path (Browser Path) because all the components on the same path .

> * This field should accept only links
=>DONE ✅

>   ![image](https://private-user-images.githubusercontent.com/83964826/321301548-87b18eed-55ed-4a27-be8b-12d5e01bf168.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDE1NDgtODdiMThlZWQtNTVlZC00YTI3LWJlOGItMTJkNWUwMWJmMTY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1Y2NjMmFjNjVkMTViN2M3NGFjMWZmOGU3YWJjYWEyY2NmMGQyNmMyYzA0ZmVhZDYzZjNlN2Q3YTdhZDk2NTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OVkwN9IS0ufDjRaFjbFcxZFTk2ThTYhn1xLt3-tqgMw)
> * Similarly, the `opportunity package` field should accept only positive numbers
=>DONE ✅

>   ![image](https://private-user-images.githubusercontent.com/83964826/321310359-98f9591c-ec2e-430d-8531-a0e1f08997cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMTAzNTktOThmOTU5MWMtZWMyZS00MzBkLTg1MzEtYTBlMWYwODk5N2NiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThkMjc2MmEyNzJjNWE5OWE1MmFiN2U1NmRhYmEwYzg5ZmIxN2U3NDk4MTcyNDEwNzcyODRhNGU1ZDVhNWRiODkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Yl-sv6QL_shJap3EeWGfxdIvxA-b08nBSd_qDO__aLc)
> * Here, make  the bg of inactive tab in the shade of light gray for light theme, it is looking good for dark theme, also the bg of should be green once it done( I know that is a bug, you have done but it not working for any reason, fix it, just mentioning here to have record )
=>DONE ✅ (I have used yellow color , which is looking bad for now because only that worked when i changed (Just to show that there is no logical issue)

>   ![Screenshot from 2024-04-10 21-05-49](https://private-user-images.githubusercontent.com/83964826/321303570-10f1b3fb-cc3f-4b30-9be6-8cb115e23de7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDM1NzAtMTBmMWIzZmItY2MzZi00YjMwLTliZTYtOGNiMTE1ZTIzZGU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1ZWFiZTFmZmI3ZWY4NDQwMDM0ZTI0M2FmMzE2MGUzYTBiZWU5ZWQwZmQzZmJkOWUzNGYwZDNmNTkyOWIxYzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.TPfFxvJZ9BMKWcTgTQBSZ-scL-nTGykWZumS-BKbEmE)
> * Both the buttons should be identical, in shape and size
=>📌Alignment & button issue is in working!! 

>   ![image](https://private-user-images.githubusercontent.com/83964826/321303676-ffb451d0-d02b-4a9e-8ba3-49b2cca014d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDM2NzYtZmZiNDUxZDAtZDAyYi00YTllLThiYTMtNDliMmNjYTAxNGQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4MjZlMDgwZTNkNjBmMmY1N2U3NjUyMDNhOWIxNDVkZTBmNjQ5YmQyMmQ3ZDM4ZjcxMzIyOTRhNjc0OTljM2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QLRxQJiP-rFS1uFYdLQjTXusc6RLabeth5xeMpiFtYM)
> * Make the buttons center alitgn for `md` and small screen size, and maybe we can make them equal in width and height or just swap the position of the two.
=>📌Alignment & button issue is in working!! 

>   ![image](https://private-user-images.githubusercontent.com/83964826/321305875-bd274ed2-842f-41ba-adae-2026bebeac99.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDU4NzUtYmQyNzRlZDItODQyZi00MWJhLWFkYWUtMjAyNmJlYmVhYzk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyMDgwMjk4ZjBhMTgzN2UwYTZlNTNhYjRiYmU0Y2Y5ZDk3NDhmNTRhOTk0NzQzYjVmZjFmZTliM2NhOWFhNGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nvi73jT8t0TGEWUUPrZ6eoBSCMLjpnqo4DaQKt5Adwg)
> * `Shaping Sucess` should come in same line, it is looking quite odd, I have added the screenshot with the dimension so that it become easier for you track .
=>📌In the previous components of our project the tagline is coming in this way that's why else i can look into it.

![Screenshot 2024-04-10 234652](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/95533641/95a9b153-b59f-4678-9c49-de6d1da0e629)

>   ![image](https://private-user-images.githubusercontent.com/83964826/321304611-5f710627-ec1d-4d91-aed6-66f0837a9cdb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDQ2MTEtNWY3MTA2MjctZWMxZC00ZDkxLWFlZDYtNjZmMDgzN2E5Y2RiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4YThkNWM1ZDJiMjc1MzcyMTcwZTA0MTgwNTc3MWI1YjJlMGZmZjFjYmZlYTE2NDU0YjdmYzA2MjliMzJhOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qKXgma5sEJNKP72RWMA7KShBIcUxXym87xPEWBNDO40)
> * Also, what if we apply `font-semibold` and text-base` in the label, ig it will look good.
=>DONE ✅

>   ![image](https://private-user-images.githubusercontent.com/83964826/321307424-0de480b5-aa1b-4a7a-86db-942710eb88d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDc0MjQtMGRlNDgwYjUtYWExYi00YTdhLTg2ZGItOTQyNzEwZWI4OGQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE1ZjYwNmVkNzdmZTcxNDI0ZjhlNWJkYTk0NTY2ZGRhZTc2MzEzMmZlMzQ2Zjg5MmJlODM5MjRiY2ZkYzY0ZmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.XRBocZI7JgHRqDJPrBH7lpawh1JoVatkStWAMN8r4ow)
>   ![image](https://private-user-images.githubusercontent.com/83964826/321307426-c23e6049-5d6e-4a06-89ca-e6d2a8c9521c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDc0MjYtYzIzZTYwNDktNWQ2ZS00YTA2LTg5Y2EtZTZkMmE4Yzk1MjFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFkZDFhNTFlNzU0MTM3MzhhZjMyMzEyNzNkNjgwNmYzMTcxM2NhYmI3NjAzMjdkMWQxYTIyZThlNTcyNjAwNTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.H7hLQjaFBwpiebTJLyZH8oJN4XyG3z1GuPvpwdfu0dk)

Sheetal-04 committed:
FIXED : OPPORTUNITY PANEL DONE (WORKING)

Sheetal-04 commented:
> * In the placeholder, in place of `Enter Opportunity Role`, make it like `e.g., Backend Developer`
>   Similarly for every input element, so that the admin got an idea about the fields
> * Remove the shadow from the input element, not looking good in pov
>   ![image](https://private-user-images.githubusercontent.com/83964826/321297870-632b026d-f506-4921-9b48-60f82e87bea1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEyOTc4NzAtNjMyYjAyNmQtZjUwNi00OTIxLTliNDgtNjBmODJlODdiZWExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyZjlkNTdkODA4NmRkNzk1M2FlYzY4OWU2YmNhZTAyNTYwZDg3MjZmYzA4MDNhN2FmZTY4NTBkZmE5MzE2N2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LE0o2GZWucoDnxeUCFCRoABLd0aUOpEH9z-PE-aJMBc)
> * Change this to `Do you want to publish this opportunity ?`
>   ![image](https://private-user-images.githubusercontent.com/83964826/321298857-cfb9d376-709d-47a0-b8c6-37308de0d926.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEyOTg4NTctY2ZiOWQzNzYtNzA5ZC00N2EwLWI4YzYtMzczMDhkZTBkOTI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNhODYyNTgzYTBkYzA2ODExMzdmNmFiMDRhNzE5MTBhOWRmYmY5ODc2MzQxOWExYWM4OTUyZjc0NjU4ODc3OTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tHaXQ3G4gwiWAPgSmyABbtUMFGjHLfy3t69Nc8ScQZM)
> * Align them properly and change the color of delete icon to red
>   ![image](https://private-user-images.githubusercontent.com/83964826/321300351-eb6715be-76b9-4250-a1c5-ef58c94baad2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDAzNTEtZWI2NzE1YmUtNzZiOS00MjUwLWExYzUtZWY1OGM5NGJhYWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU5ZTUxMjMzNGFmOTc5MDliOTY3MjAwMjhhMzg1ZjhmNTYxNzYzY2FmYjQzM2U3Yzc3NjZlYjkzYmMwNTVkNzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.z3YfIYsaZ1Le3pykV47GpuvkZ9QcDI7ANuLAd-7p59c)
> * If we navigate through the browser back button then we are redirecting to the `my-profile` page.
> * This field should accept only links
>   ![image](https://private-user-images.githubusercontent.com/83964826/321301548-87b18eed-55ed-4a27-be8b-12d5e01bf168.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDE1NDgtODdiMThlZWQtNTVlZC00YTI3LWJlOGItMTJkNWUwMWJmMTY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxMTc3OTQ2NGNjY2QyNjIxMjVlZTllNzM0NWUyMDExZWJhOTI1NGY2NTIyMjliZDkxMDU2ZDRiYzJhOTFkNWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KykUPU3SoRf-Gk4lKJOyf2IdyYJKxJAeiZLgFT5G8Bk)
> * Similarly, the `opportunity package` field should accept only positive numbers
>   ![image](https://private-user-images.githubusercontent.com/83964826/321310359-98f9591c-ec2e-430d-8531-a0e1f08997cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMTAzNTktOThmOTU5MWMtZWMyZS00MzBkLTg1MzEtYTBlMWYwODk5N2NiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0ZDQwNTVlNWE2YmQyMDg4YjM3ZWNhMzk4NmE4YTY3NmUxYzIxM2EzMjc0NjlhMjFlODEzYTViNDU0ZTRmYWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.PICsbO9Ei6ZId4TB8xDCUJ2cn3bXGfyRnHS3VG5Cbp0)
> * Here, make  the bg of inactive tab in the shade of light gray for light theme, it is looking good for dark theme, also the bg of should be green once it done( I know that is a bug, you have done but it not working for any reason, fix it, just mentioning here to have record )
>   ![Screenshot from 2024-04-10 21-05-49](https://private-user-images.githubusercontent.com/83964826/321303570-10f1b3fb-cc3f-4b30-9be6-8cb115e23de7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDM1NzAtMTBmMWIzZmItY2MzZi00YjMwLTliZTYtOGNiMTE1ZTIzZGU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5NjE2OGM5ZDZkMDA5ZWZmNDQ5NTQwZDBlZTY4N2FiYTUzMjdhYjZmMzFiNmE1ODRjNDBlODE0NjUxNjBiYjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zJ_WIQl0wUh0kK4jVRbZ774xVCu0SvSFddcRu4N8_t4)
> * Both the buttons should be identical, in shape and size
>   ![image](https://private-user-images.githubusercontent.com/83964826/321303676-ffb451d0-d02b-4a9e-8ba3-49b2cca014d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDM2NzYtZmZiNDUxZDAtZDAyYi00YTllLThiYTMtNDliMmNjYTAxNGQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJiYmRjOTAxNjA1OWE2M2QwZGE4YWRjM2U4N2UwNmZjYjJiM2FlMTA4NjQ5OTBhZDZkZGM5MjdlYWU4MDYxYzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mgyGULr43U8k6aOW96kLq8vr7WpGAVnB6_IJJj06axU)
> * Make the buttons center alitgn for `md` and small screen size, and maybe we can make them equal in width and height or just swap the position of the two.
>   ![image](https://private-user-images.githubusercontent.com/83964826/321305875-bd274ed2-842f-41ba-adae-2026bebeac99.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDU4NzUtYmQyNzRlZDItODQyZi00MWJhLWFkYWUtMjAyNmJlYmVhYzk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1YTdlY2YyZmVmY2Y0MDIwNGE2MzY4MDY1Nzg4OTEzZjRmMmRmYjFlNTRmYThmNDA2YzI4YjAzMDgxMWIyNDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Bo9Lz7sSm234cPgc1WlEMtUk-1pXXcHNf-237ZcKj8o)
> * `Shaping Sucess` should come in same line, it is looking quite odd, I have added the screenshot with the dimension so that it become easier for you track .
>   ![image](https://private-user-images.githubusercontent.com/83964826/321304611-5f710627-ec1d-4d91-aed6-66f0837a9cdb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDQ2MTEtNWY3MTA2MjctZWMxZC00ZDkxLWFlZDYtNjZmMDgzN2E5Y2RiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4ZGNhNzdhN2MwMmE2ZjFlNjg0OWRlM2JiYjk4NjQ1M2EzMDJjYzVjY2E0NWViZGUwZTkxYTE1Y2Q1M2RiMWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.r7AzvpqVVNDOvR3C2FaJaGkTGo5OqEem2z1AaK0qV7M)
> * Also, what if we apply `font-semibold` and text-base` in the label, ig it will look good.
>   ![image](https://private-user-images.githubusercontent.com/83964826/321307424-0de480b5-aa1b-4a7a-86db-942710eb88d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDc0MjQtMGRlNDgwYjUtYWExYi00YTdhLTg2ZGItOTQyNzEwZWI4OGQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdmNzY4ZTZlOWM0NmIyODQ2YTgwNGU4YzcxYWI2YzNmNjJhNmNjZDcwNTcwZjBmYjc1MGZlZjFlMDI5ZDkxYjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.k4OX4GiowSpZaJfFxWloXq6C31slSKroptRbr-ocLLo)
>   ![image](https://private-user-images.githubusercontent.com/83964826/321307426-c23e6049-5d6e-4a06-89ca-e6d2a8c9521c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDc0MjYtYzIzZTYwNDktNWQ2ZS00YTA2LTg5Y2EtZTZkMmE4Yzk1MjFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg5ZmQ2OTc2YjFkY2YxM2Q5NDgxNTdmMTM0OTQ1ZGJmY2IwZTM3NzJhNzgzN2E1ODViZmUwODA4OWQ3ZjczNzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QeFrEGHR7eOA2Mz-vHV9T8oJYvhR4VDNWx50jBdJ_QE)

* In the corrected code ,If any issue please let me know button & alignment is in working!!

Sheetal-04 commented:
> <img alt="Screenshot 2024-04-10 at 2 19 30 PM" width="393" src="https://private-user-images.githubusercontent.com/87798432/321170069-d65c4d8e-cfca-4deb-8e6c-35be6c430e24.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NzQwMzksIm5iZiI6MTcxMjc3MzczOSwicGF0aCI6Ii84Nzc5ODQzMi8zMjExNzAwNjktZDY1YzRkOGUtY2ZjYS00ZGViLThlNmMtMzViZTZjNDMwZTI0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE4Mjg1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU5MzM4NWZjNjEyODdiMWVlZjM0NjkwNDdhYWY3MzM3MmU5ZDU2NWI2NDdiYWE1OTk5ZjAwNjBhYmQ1YTE5YzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6pBObFNZdyz0WPhFuaDF5w7VShUQ2MngvDCS1g9q7pY"> i think you should remove - from OFF-LINE

=>**_DONE_**

Sheetal-04 committed:
FIXED(BUTTONS ALIGNMENT) : OPPORTUNITY PANEL DONE (WORKING)

AmanGupta2626 commented:
![Screenshot 2024-04-11 at 12 29 46 AM](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/87798432/75b75c79-a883-467a-b1d6-c5f3c087b275)
It sgould accept only +ve value and see the alignment of del icon

Sheetal-04 committed:
FIXED(Elig.Criteria ALIGNMENT) : OPPORTUNITY PANEL DONE (ALL DONE)

Sheetal-04 committed:
TITLE-TAGLINE FIXED

Sheetal-04 commented:
> > * In the placeholder, in place of `Enter Opportunity Role`, make it like `e.g., Backend Developer`
> >   =>DONE ✅
> 
> > * Similarly for every input element, so that the admin got an idea about the fields
> >   =>DONE ✅
> 
> > * Remove the shadow from the input element, not looking good in pov
> >   =>📌 Without shadow the input is not looking nice in dark theme shadow is giving kind of base to that input still for correction i have reduced the shadow from **_shadow-white/50_** to **_shadow-white/25_**
> >   ![image](https://private-user-images.githubusercontent.com/83964826/321297870-632b026d-f506-4921-9b48-60f82e87bea1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEyOTc4NzAtNjMyYjAyNmQtZjUwNi00OTIxLTliNDgtNjBmODJlODdiZWExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZkMWViM2NlNDNlYTQyZTUwOTJjZTJjMGIxYWViZmRjMjk5NTJlMTgyYmUzNTRmMzU4NDIzMjg4NDAwODA4NjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.L-6MUAvyET2kp1ZBlz0r8jjoUEafVKzubAcHnzMYlCc)
> > * Change this to `Do you want to publish this opportunity ?`
> >   =>DONE ✅
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321298857-cfb9d376-709d-47a0-b8c6-37308de0d926.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEyOTg4NTctY2ZiOWQzNzYtNzA5ZC00N2EwLWI4YzYtMzczMDhkZTBkOTI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlYzQ3MTRhNzVmMDRiODcxNDg0YTIxODI5NWNjNDNlMmRiMDA0NWVhZDYzNmVhY2ZiNzg2NTg0YjZjNWE3NDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bQcEhvfizPhYvWi7jvjA5-ansPhsxW8uGQQhXBo0hBw)
> > 
> > * Align them properly and change the color of delete icon to red
> >   =>(ICON->RED) DONE ✅ (📌Alignment & button issue is in working!! )
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321300351-eb6715be-76b9-4250-a1c5-ef58c94baad2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDAzNTEtZWI2NzE1YmUtNzZiOS00MjUwLWExYzUtZWY1OGM5NGJhYWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzYWNhNDI5NGM0YTQ5YjhlNWYwMzYwYTEyNmM5ZjdlYzc4YWYwNTU4MTU3OGQ2MjVkMjFkMWJkMmM1Y2U3ODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YN7F2qqtlkdgDmRJa-sNFxmuFe2eHb0R5I2xXpG8PA0)
> > 
> > * If we navigate through the browser back button then we are redirecting to the `my-profile` page.
> >   =>📌That is because the Back button is going back to the component which is rendering on the basis of the step &  all the components on the same path !! There is nothing which is connected to the path (Browser Path) because all the components on the same path .
> 
> > * This field should accept only links
> >   =>DONE ✅
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321301548-87b18eed-55ed-4a27-be8b-12d5e01bf168.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDE1NDgtODdiMThlZWQtNTVlZC00YTI3LWJlOGItMTJkNWUwMWJmMTY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1Y2NjMmFjNjVkMTViN2M3NGFjMWZmOGU3YWJjYWEyY2NmMGQyNmMyYzA0ZmVhZDYzZjNlN2Q3YTdhZDk2NTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OVkwN9IS0ufDjRaFjbFcxZFTk2ThTYhn1xLt3-tqgMw)
> > 
> > * Similarly, the `opportunity package` field should accept only positive numbers
> >   =>DONE ✅
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321310359-98f9591c-ec2e-430d-8531-a0e1f08997cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMTAzNTktOThmOTU5MWMtZWMyZS00MzBkLTg1MzEtYTBlMWYwODk5N2NiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThkMjc2MmEyNzJjNWE5OWE1MmFiN2U1NmRhYmEwYzg5ZmIxN2U3NDk4MTcyNDEwNzcyODRhNGU1ZDVhNWRiODkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Yl-sv6QL_shJap3EeWGfxdIvxA-b08nBSd_qDO__aLc)
> > 
> > * Here, make  the bg of inactive tab in the shade of light gray for light theme, it is looking good for dark theme, also the bg of should be green once it done( I know that is a bug, you have done but it not working for any reason, fix it, just mentioning here to have record )
> >   =>DONE ✅ (I have used yellow color , which is looking bad for now because only that worked when i changed (Just to show that there is no logical issue)
> 
> > ![Screenshot from 2024-04-10 21-05-49](https://private-user-images.githubusercontent.com/83964826/321303570-10f1b3fb-cc3f-4b30-9be6-8cb115e23de7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDM1NzAtMTBmMWIzZmItY2MzZi00YjMwLTliZTYtOGNiMTE1ZTIzZGU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1ZWFiZTFmZmI3ZWY4NDQwMDM0ZTI0M2FmMzE2MGUzYTBiZWU5ZWQwZmQzZmJkOWUzNGYwZDNmNTkyOWIxYzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.TPfFxvJZ9BMKWcTgTQBSZ-scL-nTGykWZumS-BKbEmE)
> > 
> > * Both the buttons should be identical, in shape and size
> >   =>📌Alignment & button issue is in working!!
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321303676-ffb451d0-d02b-4a9e-8ba3-49b2cca014d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDM2NzYtZmZiNDUxZDAtZDAyYi00YTllLThiYTMtNDliMmNjYTAxNGQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4MjZlMDgwZTNkNjBmMmY1N2U3NjUyMDNhOWIxNDVkZTBmNjQ5YmQyMmQ3ZDM4ZjcxMzIyOTRhNjc0OTljM2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QLRxQJiP-rFS1uFYdLQjTXusc6RLabeth5xeMpiFtYM)
> > 
> > * Make the buttons center alitgn for `md` and small screen size, and maybe we can make them equal in width and height or just swap the position of the two.
> >   =>📌Alignment & button issue is in working!!
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321305875-bd274ed2-842f-41ba-adae-2026bebeac99.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDU4NzUtYmQyNzRlZDItODQyZi00MWJhLWFkYWUtMjAyNmJlYmVhYzk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyMDgwMjk4ZjBhMTgzN2UwYTZlNTNhYjRiYmU0Y2Y5ZDk3NDhmNTRhOTk0NzQzYjVmZjFmZTliM2NhOWFhNGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nvi73jT8t0TGEWUUPrZ6eoBSCMLjpnqo4DaQKt5Adwg)
> > 
> > * `Shaping Sucess` should come in same line, it is looking quite odd, I have added the screenshot with the dimension so that it become easier for you track .
> >   =>📌In the previous components of our project the tagline is coming in this way that's why else i can look into it.
> >   =>DONE ✅ (Look into in the last deployment .Acc to me atleast till 300 that won't come into the next line)
> 
> ![Screenshot 2024-04-10 234652](https://private-user-images.githubusercontent.com/95533641/321361058-95a9b153-b59f-4678-9c49-de6d1da0e629.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3Nzg4MjAsIm5iZiI6MTcxMjc3ODUyMCwicGF0aCI6Ii85NTUzMzY0MS8zMjEzNjEwNTgtOTVhOWIxNTMtYjU5Zi00Njc4LTljNDktZGU2ZDFkYTBlNjI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE5NDg0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlYzM3YWY1ZGFhZDE1ZGI2OGNkMzhlN2JiYWM2NDZjODlkNTZjZGFiOTExMTM4MTI4NTAzNWM1OTI4Y2QzM2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.V-AMhP3mbPXhI5Q4wMv0aY13GTwZhHlpeN3c-_BZWcA)
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321304611-5f710627-ec1d-4d91-aed6-66f0837a9cdb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDQ2MTEtNWY3MTA2MjctZWMxZC00ZDkxLWFlZDYtNjZmMDgzN2E5Y2RiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4YThkNWM1ZDJiMjc1MzcyMTcwZTA0MTgwNTc3MWI1YjJlMGZmZjFjYmZlYTE2NDU0YjdmYzA2MjliMzJhOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qKXgma5sEJNKP72RWMA7KShBIcUxXym87xPEWBNDO40)
> > 
> > * Also, what if we apply `font-semibold` and text-base` in the label, ig it will look good.
> >   =>DONE ✅
> 
> > ![image](https://private-user-images.githubusercontent.com/83964826/321307424-0de480b5-aa1b-4a7a-86db-942710eb88d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDc0MjQtMGRlNDgwYjUtYWExYi00YTdhLTg2ZGItOTQyNzEwZWI4OGQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE1ZjYwNmVkNzdmZTcxNDI0ZjhlNWJkYTk0NTY2ZGRhZTc2MzEzMmZlMzQ2Zjg5MmJlODM5MjRiY2ZkYzY0ZmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.XRBocZI7JgHRqDJPrBH7lpawh1JoVatkStWAMN8r4ow)
> > ![image](https://private-user-images.githubusercontent.com/83964826/321307426-c23e6049-5d6e-4a06-89ca-e6d2a8c9521c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI3NjU5ODUsIm5iZiI6MTcxMjc2NTY4NSwicGF0aCI6Ii84Mzk2NDgyNi8zMjEzMDc0MjYtYzIzZTYwNDktNWQ2ZS00YTA2LTg5Y2EtZTZkMmE4Yzk1MjFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDEwVDE2MTQ0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFkZDFhNTFlNzU0MTM3MzhhZjMyMzEyNzNkNjgwNmYzMTcxM2NhYmI3NjAzMjdkMWQxYTIyZThlNTcyNjAwNTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.H7hLQjaFBwpiebTJLyZH8oJN4XyG3z1GuPvpwdfu0dk)

Sheetal-04 commented:
@Nandani-Paliwal Please also look into the last component of yours i have made the description  **_text-justify_** for the better alignment of the description looks good to me if not then tell me i will remove that .

Nandani-Paliwal commented on src/Components/Dashboard/OpportunityPannel/OpportunityCreation.jsx:
```diff
@@ -0,0 +1,283 @@
+								dispatch(setEditOpportunity(false), dispatch(setStep(2)));
+							}}
+							disabled={loading}
+							className={`flex w-[80%] lg:w-fit cursor-pointer items-center justify-center gap-x-2 rounded-md bg-richblack-700 hover:bg-richblack-900 py-2 px-5 text-base font-semibold text-gray-100`}
```
Ig the button will look good if change the text to `text-sm`
![Screenshot from 2024-04-11 10-42-24](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/fc15d04c-cbe3-4405-b0eb-a104d62d62fa)

here is the screenshot after changes applied that I suggested
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/68e60903-682b-4f3e-9bd4-3364ff96a9cb)

Sheetal-04 replied:
DONE

Nandani-Paliwal commented on src/Components/Dashboard/OpportunityPannel/RequirementField.jsx:
```diff
@@ -0,0 +1,105 @@
+								key={index}
+								className="flex flex-col sm:flex-row items-center text-gray-500 md:text-justify mb-4"
+							>
+								<div className="flex flex-row justify-start self-start sm:self-auto items-center gap-x-1">
```
Still, the alignment is not proper, also please change that circle SVG due its shape alignment is not looking clean, we can do something simple gray/black dot 
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/c44a57f6-5355-41bf-af81-7ed1eaececef)

We can fix this by making these changes 
- Remove the styling of `button` tag.
- We can change the colour of the SVG by editing it's `currentColor` attribute.

Here is the screenshot after applying changes that I suggested
![Screenshot from 2024-04-11 10-48-40](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/4503ab82-da16-4da8-9ca5-2199649a30cf)

Sheetal-04 replied:
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/95533641/a957488a-ac49-443d-bd91-e569cb60cd34)
 Actually i am thinking to put the delete button at the end of description because you are now only putting 3 to 4 words but  when in the criteria there is 2 to 3 line para and in small screen that is not looking nice .if you guys are ohk with this then i can make the changes .see when i put it at end

Sheetal-04 replied:
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/95533641/5c3d9367-c8e7-432e-ade8-fa854dd8a08f)
Look into it looking more accurate when the delete icon is at the end

Sheetal-04 replied:
Same i am keeping the bullet point at start because when the content is more that should come at start else that will look at the center of the description which will not look good .

Sheetal-04 committed:
Fixes
